lib/modules/manager/velaci/extract.ts

```diff
@@ -0,0 +1,59 @@
+import { load } from 'js-yaml';
+import { logger } from '../../../logger';
+import { getDep } from '../dockerfile/extract';
+import type { PackageDependency, PackageFile } from '../types';
+import type { VelaPipelineConfiguration } from './types';
+
+export function extractPackageFile(file: string): PackageFile | null {
+  let doc: VelaPipelineConfiguration | null;
```

Comment:
```suggestion
  let doc: VelaPipelineConfiguration | undefined;
```

Reply:
ah yes, i was thinking of the function return 🤦🏼